src/test/java/gov/healthit/chpl/aqa/pageObjects/BasePage.java

```diff
@@ -21,4 +21,13 @@ public static WebElement mainContent(final WebDriver driver) {
         element = driver.findElement(By.id("main-content"));
         return element;
     }
+
+    /**
+     * Return CQM Accordion on listing details page or edit CP page.
+     * @param driver WebDriver
+     * @return CQM Accordion element
+     */
+    public static WebElement cqmAccordion(final WebDriver driver) {
```

Comment:
I don't think this should be part of the BasePage object. Not every page has this element, while they all should have the "main-content" one. My guess is that the only things in the basepage should be that main-content and navigation header/footer stuff